packages/styles/scss/components/pricing-table/_pricing-table.scss

```diff
@@ -0,0 +1,225 @@
+//
+// Copyright IBM Corp. 2022
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import 'carbon-components/scss/components/tag/tag';
+@import '../../globals/imports';
+@import '../../globals/vars';
+@import '../buttongroup/buttongroup';
+@import '../structured-list/structured-list';
+
+@include structured-list;
+@include tags;
+
+@mixin pricing-table {
+  .#{$prefix}--pricing-table,
+  :host(#{$dds-prefix}-pricing-table) {
+    @extend .#{$dds-prefix}--structured-list;
+  }
+
+  .#{$prefix}--pricing-table-group,
+  :host(#{$dds-prefix}-pricing-table-group) {
+    @extend .#{$prefix}--structured-list-group;
+  }
+
+  .#{$prefix}--pricing-table-header-row,
+  :host(#{$dds-prefix}-header-row) {
+    @extend .#{$prefix}--structured-list-header-row;
+  }
+
+  .#{$prefix}--pricing-table-header-row,
+  .#{$prefix}--pricing-table-row,
+  .#{$prefix}--pricing-table-group tr,
+  :host(#{$dds-prefix}-pricing-table-header-row),
+  :host(#{$dds-prefix}-pricing-table-row),
+  :host(#{$dds-prefix}-pricing-table-group) tr {
+    @extend .#{$prefix}--structured-list-all-rows;
+
+    position: relative;
+
+    &::after {
+      content: '';
+      display: block;
+      position: absolute;
+      right: 0;
+      bottom: 0;
+      width: 100%;
+      height: 1px;
+      background-color: $ui-03;
+      box-shadow: 50vw 0 $ui-03, 100vw 0 $ui-03, 150vw 0 $ui-03, 200vw 0 $ui-03,
```

Comment:
Did you not find a way to get the sass function working?

Reply:
I couldn't figure it out quickly, so I shifted priority to getting this PR into a reviewable state. I'll look into it more today.